.github/workflows/publish.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish
+on:
+  release:
+    types: [released]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: curl -L ${{ github.event.release.tarball_url }} > source.tar.gz
+      - uses: filecoin-station/publish-zinnia-module-action@v0
+        with:
+          source: source.tar.gz
```

Comment:
Nitpick: should we use the repository name as the filename instead of "source.tar.gz"? This would be meaningful if the filename were preserved in IPFS metadata by w3up.

Reply:
Ha! I just wanted to verify this, and see, our uploads are completely broken :D

https://bafkreifsrsklegk4r3jft4fucwvo4pzzwczjecsfg5qrjgp2arevnel2ee.ipfs.w3s.link/

<img width="146" alt="Screenshot 2024-01-31 at 11 19 40" src="https://github.com/filecoin-station/spark/assets/10247/2f2de73f-5cec-4af0-8d64-60dd9edf4df2">

Reply:
I fixed the action. I also checked out an upload and nothing of the file name is used for meta information